refinery/analysis_manager/tasks.py

```diff
@@ -473,7 +473,11 @@ def _get_galaxy_download_task_ids(analysis):
     tool.create_analysis_output_node_connections()
     galaxy_instance = analysis.workflow.workflow_engine.instance
     try:
-        download_list = tool.get_galaxy_dataset_download_list()
+        download_list = tool_manager.models.AnalysisNodeConnection.\
+            objects.filter(
+                is_refinery_file=True, analysis=analysis,
+                direction=OUTPUT_CONNECTION
+            )
```

Comment:
@hackdna I'm not familiar with the history of this enough to make this call, but the get_galaxy_dataset_download_list() function is used nowhere else.  can it be removed?  would it be better to replace it with the output of the the AnalysisNodeConnection.objects.filter() operation (maybe along with some checking to make sure the list is non-empty)?

Reply:
i use the term "filter" loosely to just mean that we do some subsetting and transformation

Reply:
OK but I was referring to the replacement code here which is just filtering (vs a bunch of operations in `get_galaxy_dataset_download_list()`). If this is equivalent you can remove the function. Another question is: will this help you with refactoring later?

Reply:
likely, yes. it depends on how i refactor, but i assume this replacement will be helpful. removing the get_galaxy_dataset_download_list() sped things up considerably. in this case i'll remove the function altogether.